samples/CommunityToolkit.Maui.Sample/Pages/Base/BaseGalleryPage.cs

```diff
@@ -25,7 +25,9 @@ protected BaseGalleryPage(string title, IDeviceInfo deviceInfo, TViewModel viewM
 
 	async void HandleSelectionChanged(object? sender, SelectionChangedEventArgs e)
 	{
-		ArgumentNullException.ThrowIfNull(sender);
+		try
```

Comment:
do we need try/catch here?

Reply:
Oh no I must have forgotten to remove that. Good spot!